src/controller/main.cpp

```diff
@@ -19,6 +19,8 @@ bool is_autotuning = false;                   // Flag for whether we are in auto
 unsigned long lastCycleStart = 0;             // Tracks the start time of the pump cycle
 float flowPercentage = 0;                     // Declare flowPercentage with an initial value
 unsigned long lastTempUpdate = 0;
+bool brew_button_state;
```

Comment:
set these to true on default, the pins are pulled high. or initialize them with a digitalRead in setup